src/index.js

```diff
@@ -1,16 +1,27 @@
 import createTest, { isTestLibrarySupported } from './create-test';
 
+const _returnChildState = (stateKey, stateObj) => {
+  if (!stateKey) {
+    return stateObj;
+  };
+  const splitState = stateKey.split('.');
+  return splitState.reduce((higherLeveLobject, levelLabel) => {
+    const currentLevelObject = higherLeveLobject[levelLabel]
+    return currentLevelObject === undefined ? {} : currentLevelObject;
```

Comment:
I think instead of just defaulting we should throw an error indicating that they have passed in an invalid stateKey